src/sliceDataStorage.cpp

```diff
@@ -560,9 +560,14 @@ Polygons SliceDataStorage::getMachineBorder(int checking_extruder_nr) const
 
     Polygons disallowed_areas = mesh_group_settings.get<Polygons>("machine_disallowed_areas");
     disallowed_areas = disallowed_areas.unionPolygons(); // union overlapping disallowed areas
-    for (PolygonRef poly : disallowed_areas)
-        for (Point& p : poly)
-            p = Point(machine_size.max.x / 2 + p.X, machine_size.max.y / 2 - p.Y); // apparently the frontend stores the disallowed areas in a different coordinate system
+
+    // The disallowed areas are always expressed in buildplate-centered coordinates
```

Comment:
I was a bit confused by the wording of this comment; you can interpret it that that the coordinates we get from cura are always expressed in buildplate-centered coords, or if they should be expressed as buildplate-centered coordinates.

I think if we change the wording a little it makes a little bit more clear.
```suggestion
    // Make sure the disallowed areas are expressed in buildplate-centered coordinates
```